Tests/Hotels/TcSprint2.feature

```diff
@@ -0,0 +1,43 @@
+@TC-Sprint-2
+Feature: TC-Sprint-2
+
+  @TC-31
+  Scenario: Verify language can be changed successfully
+    When I Lunch hotels.com homepage
+    Then I Click on “English“ language
```

Comment:
Then => When

Reply:
Pls correct ALL Gherkin labels for all tests.
Precondition for test => Given
Action step => When
Verification step => Then
Any repeated Given/When/Then should be replaced by And.